esp/src/esp.ino

```diff
@@ -43,6 +47,13 @@ void setup() {
   Serial.setTimeout(100);
   setup_wifi();
 
+  configTime(1 * 3600, 0, "pool.ntp.org", "time.nist.gov"); //time zone is GMT + 1
```

Comment:
What if we use module in LAN without access to the internet? Maybe there should be possibility to skip waiting for time? Of course at cost of not working TimeSensor. TimeSensor could send undefined value if there is no access to time service.